charts/tigera-operator/values.yaml

```
@@ -58,4 +58,9 @@ calicoctl:
   image: docker.io/calico/ctl
   tag: master
 
-kubeletVolumePluginPath: /var/lib/kubelet
\ No newline at end of file
+kubeletVolumePluginPath: /var/lib/kubelet
+
+#for windows or eBFP support it is necessary to supply the api server
+kubernetesServiceEndpoint:
+  host: "" #e.g. k8s.mycompany.com
```

```suggestion
  host: ""
```

We can remove the example here